graylog2_handler/README.rst

```diff
@@ -0,0 +1,17 @@
+GrayLog2 GELF log handler
+=========================
```

Comment:
can you update the README to follow the OCA template (https://github.com/OCA/maintainer-tools/tree/master/template/module)